src/main/java/org/dependencytrack/resources/v1/VulnerabilityResource.java

```diff
@@ -618,4 +623,58 @@ public Response unassignVulnerability(@ApiParam(value = "The UUID of the vulnera
             }
         }
     }
+
+    @POST
+    @Path("/tags")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(
+            value = "Updates tags for a vulnerability"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 401, message = "Unauthorized"),
+            @ApiResponse(code = 404, message = "The vulnerability could not be found")
+    })
+    @PermissionRequired(Permissions.Constants.VULNERABILITY_MANAGEMENT)
+    public Response updateVulnerabilityTags(Vulnerability jsonVuln) {
```

Comment:
It's a bit odd to accept an entire `Vulnerability` object, but only using two fields of it. From the outside, it will not be possible to tell which fields are used and which are not. Can we trim this down a bit?

For example:

* Change the path to `/{uuid}/tag`
* Accept a list of `String` instead of `Vulnerability` as payload